lib/pubsub/topic.js

```diff
@@ -144,11 +147,17 @@ Topic.prototype.publish = function(messages, callback) {
   callback = callback || util.noop;
 
   var body = {
-    topic: this.name,
     messages: messages.map(Topic.formatMessage_)
   };
 
-  this.makeReq_('POST', 'topics/publishBatch', null, body, callback);
+  var path = this.name + ':publish';
```

Comment:
Is `:publish` right ? For a URL path? If this isn't a typo, doesn't that seem off for an API URL?